graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -189,25 +192,30 @@ def __init__(self, renamings, query_type, scalar_types):
         self.builtin_types = frozenset({'String', 'Int', 'Float', 'Boolean', 'ID'})
 
     def _rename_name_and_add_to_record(self, node):
-        """Rename the value of the node, and add the name pair to reverse_name_map.
+        """Return a copy of input node with a new name, and add the name pair to reverse_name_map.
```

Comment:
nit: Technically, this is inaccurate – we only return a copy if changes were necessary. I'd avoid promising a copy or non-copy here, and just describe the change that is applied (if necessary): `Change the name of the input node if necessary, and add the name pair to reverse_name_map` or something to that effect, similar to the documentation in `ast_rewriting.py`.